tests/Unit/Http/HeadersTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DDTrace\Tests\Unit\Http;
+
+use DDTrace\Http\Headers;
+use PHPUnit\Framework;
+
+
+final class HeadersTest extends Framework\TestCase
+{
+    public function test_headersMapToColonSeparatedValues_emptyArray()
```

Comment:
I'm OK with snake case for test names, but it seems a bit harder to read with both snake case and camel case combined. Could we change this to be all snake case or all camel case? :)

Reply:
You know? For consistency with all the other tests that we have, let's revert back to camel case!